### PR TITLE
Fixed 3 issues

### PR DESCRIPTION
**Pull request by johne99**
Fixed throw in exception
Did Random field
Changed =- to -=

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -18,6 +18,7 @@ public class TorpedoStore {
   private double FAILURE_RATE = 0.0; //NOSONAR
 
   private int torpedoCount = 0;
+  private Random generator = new Random();
```
Comment (to-dor) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:21]: Szép munka

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -35,18 +36,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (to-dor) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:39]: Izgalmas megoldás

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 3]
```diff
@@ -35,18 +36,17 @@ public TorpedoStore(int numberOfTorpedos){
...
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
     double r = generator.nextDouble();
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```
Comment (to-dor) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:49]: Gyönyörű!

**Review by to-dor — CHANGES_REQUESTED** [review]

**Comment by johne99** [comment]
Finished fixes

**Pull request merged**